modules/siddhi-core/src/main/java/io/siddhi/core/table/record/RecordStoreCompiledCondition.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.siddhi.core.table.record;
+
+import io.siddhi.core.config.SiddhiAppContext;
+import io.siddhi.core.config.SiddhiQueryContext;
+import io.siddhi.core.executor.ExpressionExecutor;
+import io.siddhi.core.util.collection.operator.CompiledCondition;
+
+import java.util.Map;
+
+/**
+ * Compiled condition of the {@link AbstractRecordTable}
+ */
+public class RecordStoreCompiledCondition implements CompiledCondition {
```

Comment:
We can keep `RecordStoreCompiledCondition` in the place this was earlier there, if I'm not wrong?

Reply:
Fixed in 93a318b